### ISSUE TITLE
Sequel Ace crashes after selecting database (with 1+ VIEWs) on MariaDB

### ISSUE THREAD

**Issue by mgoldbeck** [Sequel-Ace/Sequel-Ace#79]
- Sequel Ace Version: 2.0.2
- macOS Version: 10.15.4
- MySQL Version: mysql  Ver 15.1 Distrib 10.4.12-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2

**Description**
Selecting a database by the DB selector, Sequel Aces crashes

**Steps To Reproduce**
1. open connection
2. select database

**Expected Behaviour**
Being able to select a database

Application Specific Information:
```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[NSNull isEqualToString:]: unrecognized selector sent to instance 0x7fff90413c00'
abort() called
terminating with uncaught exception of type NSException
```

**Comment by gboudreau** [comment]
The only way I can reproduce is if I select the `mysql` database; all my own database can be selected without problems...
Are you noticing the same behaviour?

**Comment by gboudreau** [comment]
Ah, it's about VIEWs. If your DB has views, it will crash. It's because MariaDB returns different columns in the response of `SHOW TABLE STATUS` that MySQL.

**Comment by Jason-Morcos** [comment]
Please try 2.1.0 beta 3 and let us know if it fixed the issue, @mgoldbeck! https://github.com/Sequel-Ace/Sequel-Ace/releases/tag/2.1.0-beta3

**Comment by mgoldbeck** [comment]
@Jason-Morcos sorry for late response. Tested it today with the released 2.1.1. Work now :)

btw. I like the new logo much better 👍
